.golangci.yaml

```diff
@@ -1,5 +1,23 @@
-# https://golangci-lint.run/usage/configuration/
+version: "2"
 linters:
   enable:
     - makezero
     - misspell
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+formatters:
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
Did you copy this config from somewhere else? Because we only have the `examples` directory https://github.com/urfave/cli/tree/main/examples

Since our examples code are minimal, I think we should be able to include them in the linter too and not cause any errors. What do you think?

Reply:
This was auto-generated by golangci-lint migrate